playbooks/tests/tasks/rally/bbc-cloud-validate.yml.tmpl

```diff
@@ -0,0 +1,409 @@
+      runner:
+        type: "constant"
+        times: 50
+        concurrency: 5
+      context:
+        users:
+          tenants: 2
+          users_per_tenant: 3
+      sla:
+        failure_rate:
+          max: 5
+
+  NeutronNetworks.create_and_delete_networks:
+    -
+      args:
+        network_create_args: {}
+      runner:
+        type: "constant"
+        times: 20
+        concurrency: 1
+      context:
+        users:
+          tenants: 1
+          users_per_tenant: 1
+        quotas:
+          neutron:
+            network: -1
+      sla:
+        failure_rate:
+          max: 5
+
+  NeutronNetworks.create_and_delete_ports:
+    -
+      args:
+        network_create_args: {}
+        port_create_args: {}
+        ports_per_network: 5
+      runner:
+        type: "constant"
+        times: 100
+        concurrency: 5
+      context:
+        users:
+          tenants: 1
+          users_per_tenant: 1
+        quotas:
+          neutron:
+            network: -1
+            port: -1
+      sla:
+        failure_rate:
+          max: 5
+
+  NeutronNetworks.create_and_delete_subnets:
+    -
+      args:
+        network_create_args: {}
+        subnet_create_args: {}
+        subnet_cidr_start: "1.1.0.0/30"
+        subnets_per_network: 2
+      runner:
+        type: "constant"
+        times: 20
+        concurrency: 1
+      context:
+        users:
+          tenants: 1
+          users_per_tenant: 1
+        quotas:
+          neutron:
+            network: -1
+            subnet: -1
+      sla:
+        failure_rate:
+          max: 5
+
+  CinderVolumes.create_and_delete_volume:
+    -
+      args:
+        size: 1
+      runner:
+        type: "constant"
+        times: 20
+        concurrency: 5
+      context:
+        users:
+          tenants: 3
+          users_per_tenant: 2
+        quotas:
+          cinder:
+            volumes: -1
+      sla:
+        failure_rate:
+          max: 5
+
+  CinderVolumes.create_and_delete_snapshot:
+    -
+      args:
+          force: false
+      runner:
+        type: "constant"
+        times: 20
+        concurrency: 5
+      context:
+        users:
+          tenants: 3
+          users_per_tenant: 2
+        volumes:
+          size: 1
+      sla:
+        failure_rate:
+          max: 5
+
+  CinderVolumes.create_from_volume_and_delete_volume:
+    -
+      args:
+        size: 1
+      runner:
+        type: "constant"
+        times: 20
+        concurrency: 5
+      context:
+        users:
+          tenants: 3
+          users_per_tenant: 2
+        volumes:
+          size: 1
+      sla:
+        failure_rate:
+          max: 5
+
+  NovaServers.boot_and_delete_server:
+    -
+      args:
+        flavor:
+            name: "m1.tiny"
+        image:
+            name: @@IMAGE_NAME@@
+        force_delete: false
+        nics:
+          - { "net-id": "@@NET_ID@@" }
+      runner:
+        type: "constant"
+        times: 100
+        concurrency: 12
```

Comment:
But concurrency 12, I think that means it'll run this 12 at a time, each thread booting one.